pkg/agent/core/ngt/service/vqueue/queue.go

```diff
@@ -291,9 +273,9 @@ func (v *vqueue) addDelete(d key) {
 	v.dmu.Unlock()
 }
 
-func (v *vqueue) flushAndLoadInsert() (uii []index) {
+func (v *vqueue) flushAndRangeInsert(f func(uuid string, vector []float32) bool) {
```

Comment:
Should we pass `context.Context` ?
And if the context is canceled, should we return in L.295?

Reply:
I think we shouldn't stop the createIndex process when the context is canceled.
It is because if we stop createIndex, the index may be lost due to uncompleted process of vqueue.